eng/configurations/targetgroups.props

```diff
@@ -122,18 +122,23 @@
       <CompatibleWith>netstandard</CompatibleWith>
     </TargetGroups>
     <TargetGroups Include="netcoreapp3.0">
-      <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
       <TargetFramework>netcoreapp3.0</TargetFramework>
       <Imports>netcoreapp2.2</Imports>
       <CompatibleWith>netstandard2.1</CompatibleWith>
     </TargetGroups>
-    <!-- netcoreapp is an alias for netcoreapp3.0 -->
-    <TargetGroups Include="netcoreapp">
-      <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
-      <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetGroups Include="netcoreapp5.0">
+      <NETCoreAppMaximumVersion>5.0</NETCoreAppMaximumVersion>
+      <TargetFramework>netcoreapp5.0</TargetFramework>
       <Imports>netcoreapp3.0</Imports>
```

Comment:
This will need to change to `netcoreapp3.1` when we make that TFM, right?

Reply:
yes right